instrumentation/clickhouse/clickhouse-client-common/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,15 @@
+plugins {
+  id("otel.java-conventions")
+  id("otel.javaagent-instrumentation")
+}
+
+repositories {
+  mavenCentral()
```

Comment:
is this needed?

Reply:
@jaydeluca it doesn't need, it should be removed.